crates/prql-compiler/src/semantic/resolver/transforms.rs

```diff
@@ -1068,4 +1067,28 @@ mod tests {
             - Wildcard
         "###);
     }
+
+    #[test]
+    fn test_invalid_lineage_in_transform() {
+        let result = parse_resolve_and_lower(
+            "
+        from tbl
+        group id (
+            sort -val
+        )
+        ",
+        );
+        assert!(result.is_err());
```

Comment:
Could we move this into `test_bad_error_messages.rs` and snapshot the error message? (If there's no difference between the examples, fine to keep only one)